tests/management/workspace/test_view.py

```diff
@@ -410,9 +426,12 @@ def test_delete_workspace(self):
 
         url = reverse("workspace-detail", kwargs={"uuid": workspace.uuid})
         client = APIClient()
-        response = client.delete(url, None, format="json", **self.headers)
+        test_headers = self.headers.copy()
+        test_headers["HTTP_ACCEPT"] = "application/problem+json"
```

Comment:
Look at adding a v2 identity request which inherits from v1 to add headers.

Reply:
One interesting thing about this I guess is that the headers are dependent on the request. So like if we added both accept media types unconditionally, I guess we wouldn't have reproduced the bug.

I wonder if we should try and use the generated client in the tests somehow? (If so this is very likely a future reliability topic I guess)

Reply:
@alechenninger yeah, good point. 

that's actually something I've brought up in the past for PR check/IQE tests, around potentially building clients on PR changes and having a small suite that would test client changes based on spec changes and run through some tests.

Reply:
Added a shell ticket here: https://issues.redhat.com/browse/RHCLOUD-35416